QQE-378.md

```diff
@@ -0,0 +1,44 @@
+# QQE-378 - HTTP module / REST use-cases - Development topic compression Brotli4j
+## Links
+Jira : [QQE-378](https://issues.redhat.com/browse/QQE-378)
+
+PR:  TODO
+## Scope of the testing
+ - Integration of the Brotli4J compression library with Quarkus, particularly in the context of REST endpoints using RestEasy Reactive.
+ - The test coverage needs to be defined for both JVM and native mode.
+ - To use Brotli4J compression we will need a custom Server configuration
+   * Utilize Vert.x http server options  to enable compression support.
+   * Include the compressor using Brotli4J options compression.
+   * Ensure that the server is configured to respect the Accept-Encoding header to automatically compress the response body with one of the supported compressions and send it back to the client.
+### Test scenarios 
+
+- Compression Enabled, Response Compressed
+   *    Set up a Quarkus application with the Brotli4J compression library
+   *    Configure the HTTP server to enable compression using io.vertx.core.http.HttpServerOptions
```

Comment:
Same question as above, why is that necessary? Shouldn't you configure Quarkus instead?

Reply:
As far as I know, Quarkus itself is not capable of managing brotli4J but yes gzip. FOr that reason, vert.x configuration HTTP server options becomes relevant in this context.